docs/advanced.md

```diff
@@ -9,12 +9,17 @@ all outgoing requests.
 !!! hint
     A Client instance is equivalent to a Session instance in `requests`.
```

Comment:
Is this also to be `AsyncClient`?

Reply:
Well, here (and elsewhere on this page) "Client" refers to the general idea of a class for making requests — sync or async.

It's a bit blurry at the moment since we we don't have anything else than `AsyncClient` yet, but I'd say for now it doesn't look *too* unnatural?